packages/aw-signer/package.json

```diff
@@ -29,14 +30,17 @@
     "@types/jest": "^29.5.14",
     "@types/node-localstorage": "^1.3.3"
   },
-  "type": "commonjs",
+  "type": "module",
   "main": "./dist/src/index.js",
   "types": "./dist/src/index.d.ts",
   "typings": "./dist/src/index.d.ts",
-  "files": [
-    "dist",
-    "!**/*.tsbuildinfo"
-  ],
+  "exports": {
+    ".": {
+      "types": "./dist/src/index.d.ts",
+      "import": "./dist/src/index.js",
+      "require": "./dist/src/index.cjs"
+    }
+  },
```

Comment:
This means we are only exporting the `index.js` and not the rest of the js files in the `dist` dir?